service/user.service.js

```diff
@@ -72,11 +74,17 @@ const getAllUsers = async () => {
 
 const getUser = async (email) => {
     const user = await UserModel.findOne({ email });
-    return new UserDTO(user);
+    const userData = new UserDTO(user);
+    const file = await fs.readFileSync(path.resolve(__dirname, "../uploads/avatars", user.avatar));
```

Comment:
@ikadushkina use async fs analogue for writing files, as the sync operation will block the event loop. Btw, currently the operation is sync, so the await is useless here.